std/array.d

```diff
@@ -3339,6 +3340,69 @@ if (isDynamicArray!A)
     app.put(r[]);
 }
 
+unittest // Issue 13300
```

Comment:
For better or worse, all `unittest` blocks are supposed to be marked with `@safe` or `@system`. So, if the tests work with `@safe`, then the `unittest` block should be marked as `@safe`, and if they don't, then it should be marked as `@system`.